src/features/profile/pages/SuspendAccountConfirmation/SuspendAccountConfirmation.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useCallback } from 'react'
+import { View } from 'react-native'
+import styled from 'styled-components/native'
+
+import { navigateToHome } from 'features/navigation/helpers'
+import { useEmailUpdateStatus } from 'features/profile/helpers/useEmailUpdateStatus'
+import { ButtonPrimary } from 'ui/components/buttons/ButtonPrimary'
+import { ButtonTertiaryBlack } from 'ui/components/buttons/ButtonTertiaryBlack'
+import { GenericInfoPageWhite } from 'ui/pages/GenericInfoPageWhite'
+import { BicolorUserError } from 'ui/svg/BicolorUserError'
+import { Clear } from 'ui/svg/icons/Clear'
+import { Spacer, Typo } from 'ui/theme'
+import { DOUBLE_LINE_BREAK } from 'ui/theme/constants'
+
+export function SuspendAccountConfirmation() {
+  const { data: emailUpdateStatus } = useEmailUpdateStatus()
+
+  const onClose = useCallback(() => {
+    navigateToHome()
+  }, [])
+
+  const handleSuspendAccount = useCallback(() => {
+    // TODO(PC-22601): add suspend account API call
+  }, [])
+
+  if (!emailUpdateStatus || emailUpdateStatus?.expired) {
+    navigateToHome()
+  }
```

Comment:
Le `emailUpdateStatus` sera à `undefined` avant que le fetch soit fait ?

```suggestion
  useEffect(() => {
    if (!isLoading && (!emailUpdateStatus || emailUpdateStatus?.expired)) {
      navigateToHome()
    }
  }, [...])
```

ce serait plus correct non ?

Reply:
Mais du coup, s'il est à `undefined`, il va te rediriger sur la home avant même d'avoir fini de fetcher nan ?

Et le `useEffect` est presque requis à ce niveau. Même si ça fonctionne comme ça... Parce que tu auras peut-être plusieurs rendus, et réexécuter ce bloc pour rien n'est pas utile.

Reply:
Non il ne redirige pas sur la home pour autant, ça va sûrement trop vite.
Ok je le rajoute.